pom.xml

```diff
@@ -113,7 +124,15 @@
         <dependency>
             <groupId>sc.fiji</groupId>
             <artifactId>bigwarp_fiji</artifactId>
+
+			<!-- See Issue https://github.com/saalfeldlab/bigwarp/issues/83
+			<version>5.1.0</version>-->
         </dependency>
+		<dependency>
+			<groupId>net.imglib2</groupId>
+			<artifactId>imglib2</artifactId>
+			<version>5.8.0</version>
```

Comment:
For best compatibility with other projects in the SciJava ecosystem, please don't override the version like this.

Instead, change the version property defined in the `<properties>` section:

    <imglib2.version>5.8.0</imglib2.version>

---

The current `master` branch of `pom-scijava` already defines this version, so the version declaration here can be removed once `pom-scijava-28.0.0` is released. See [`pom-scijava/pom.xml#L340-L341`](https://github.com/scijava/pom-scijava/blob/a78ae8631fccaba7791c3b918c6f939dffab0b65/pom.xml#L340-L341)

Reply:
Is there a way to do this for `<artifactId>scijava-ui-swing</artifactId>` as well ?

And thanks a lot for your comments, @imagejan !

Reply:
Yes, it should be done for *all* versions of *all* artifacts managed by `pom-scijava`. The general pattern is `<{artifactId}.version>x.x.x</{artifactId}.version>`.

To see if a component is managed by `pom-scijava`, look in its [`pom.xml`](https://github.com/scijava/pom-scijava/blob/master/pom.xml).

Reply:
Thanks, that's changed in https://github.com/bigdataviewer/bigdataviewer-playground/pull/51/commits/e0d91c9b061a726cfde4c96c74be220facada629